services/QuillLMS/app/controllers/teacher_infos_controller.rb

```diff
@@ -28,6 +28,20 @@ def update
     }, status: 200
   end
 
+  def request_teacher_verification
+    admin_ids = request_teacher_verification_params[:admin_ids]
+    message = request_teacher_verification_params[:message]
```

Comment:
Beyond the scope of this PR, but it's worth flagging that by not persisting `message` anywhere we make it impossible for a different admin to view the list of all pending requests with their messages.  We don't need to do that right now, but it's worth having on our radar as we consider expanding permissions management.